### PR TITLE
clinic: add PingCAP Clinic Diagnostic Data

### PR DESCRIPTION
**Pull request by Oreoxmt**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/1819 https://github.com/pingcap/docs-tidb-operator/pull/1875
- Other reference link(s):

### CODE REVIEW

File: en/clinic-data-instruction.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+| :------ | :------ |:-------- |
+| Performance data | `cpu_profile.proto`, `mem_heap.proto`, `goroutine.txt`, `mutex.txt` | `collectors:perf` (Diag does not collect this data type by default) |
+| Real-time configuration | `config.json` | `collectors:config` |
+
+## TiKV diagnostic data
+
+| Data type | Exported file | Parameter for data collection by PingCAP Clinic |
+| :------ | :------ |:-------- |
+| Real-time configuration | `config.json` | `collectors:config` |
+| Performance data | `cpu_profile.proto` | `collectors:perf` (Diag does not collect this data type by default) |
```
Comment (Oreoxmt) [inline en/clinic-data-instruction.md:33]:
```suggestion
```

File: en/clinic-data-instruction.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+## PD diagnostic data
+
+| Data type | Exported file | Parameter for data collection by PingCAP Clinic |
+| :------ | :------ |:-------- |
+| Real-time configuration | `config.json` |`collectors:config` |
+| Outputs of the command `tiup ctl pd -u http://${pd IP}:${PORT} store` | `store.json` | `collectors:config` |
+| Outputs of the command `tiup ctl pd -u http://${pd IP}:${PORT} config placement-rules show` | `placement-rule.json` | `collectors:config` |
+| Performance data | `cpu_profile.proto`, `mem_heap.proto`, `goroutine.txt`, `mutex.txt` | `collectors:perf` (Diag does not collect this data type by default) |
```
Comment (Oreoxmt) [inline en/clinic-data-instruction.md:42]:
```suggestion
```

File: en/clinic-data-instruction.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+## TiFlash diagnostic data
+
+| Data type | Exported file | Parameter for data collection by PingCAP Clinic |
+| :------ | :------ |:-------- |
+| Real-time configuration | `config.json` |`collectors:config` |
+| Performance data | `cpu_profile.proto` | `collectors:perf` (Diag does not collect this data type by default)  |
```
Comment (Oreoxmt) [inline en/clinic-data-instruction.md:49]:
```suggestion
```

File: en/clinic-data-instruction.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+| Data type | Exported file | Parameter for data collection by PingCAP Clinic |
+| :------ | :------ |:-------- |
+| Basic information of the cluster, including the cluster ID | `cluster.json` | The data is collected per run by default. |
+| Detailed information of the cluster | `tidbcluster.json` | The data is collected per run by default. |
+
+## TiDB diagnostic data
+
+| Data type | Exported file | Parameter for data collection by PingCAP Clinic |
+| :------ | :------ |:-------- |
+| Performance data | `cpu_profile.proto`, `mem_heap.proto`, `goroutine.txt`, `mutex.txt` | `collectors:perf` (Diag does not collect this data type by default) |
```
Comment (Oreoxmt) [inline en/clinic-data-instruction.md:25]:
```suggestion
```

File: en/clinic-data-instruction.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+This document provides the types of diagnostic data that can be collected by PingCAP Clinic Diagnostic Service (PingCAP Clinic) from the TiDB clusters deployed using TiDB Operator. Also, the document lists the parameters for data collection corresponding to each data type. When running a command to [collect data using Diag client (Diag)](clinic-user-guide.md), you can add the required parameters to the command according to the types of data to be collected.
+
+The diagnostic data collected by PingCAP Clinic is **only** used for troubleshooting cluster problems.
+
+Clinic Server is a diagnostic service deployed in the cloud. Currently, you can upload the collected diagnostic data to Clinic Server China only. The uploaded data is stored in the AWS S3 China (Beijing) region server set up by PingCAP. PingCAP strictly controls permissions for data access and only allows authorized in-house technical support staff to access the uploaded data.
+
+## TiDB cluster information
+
+| Data type | Exported file | Parameter for data collection by PingCAP Clinic |
```
Comment (ran-huang) [inline en/clinic-data-instruction.md:16]: I think this is more concise.
```suggestion
| Data type | Exported file | Parameter for PingCAP Clinic |
```

File: en/clinic-data-instruction.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+| :------ | :------ |:-------- |
+| Basic information of the cluster, including the cluster ID | `cluster.json` | The data is collected per run by default. |
+| Detailed information of the cluster | `tidbcluster.json` | The data is collected per run by default. |
+
+## TiDB diagnostic data
+
+| Data type | Exported file | Parameter for data collection by PingCAP Clinic |
```
Comment (ran-huang) [inline en/clinic-data-instruction.md:23]: ditto

**Comment by Oreoxmt** [comment]
Merge this PR into https://github.com/pingcap/docs-tidb-operator/pull/1890

**Pull request closed without merging**